lib/klaviyo.ts

```diff
@@ -5,14 +5,32 @@ interface EmailServiceProvider {
 const Klaviyo = (token: string): EmailServiceProvider => ({
   send: async (event, recipient, context) => {
     const formParams = new URLSearchParams();
-    formParams.append("data", JSON.stringify({
-      token,
-      event,
-      customer_properties: { $email: recipient },
-      properties: context,
-    }));
+    formParams.append(
+      "data",
+      JSON.stringify({
+        token,
+        event,
+        customer_properties: { $email: recipient },
+        properties: context,
+      })
+    );
 
-    await fetch("https://a.klaviyo.com/api/track", { method: "POST", body: formParams });
+    console.log(
+      "Klaviyo request: https://a.klaviyo.com/api/track, ",
+      formParams
+    );
+
+    const response = await fetch("https://a.klaviyo.com/api/track", {
```

Comment:
We could return the response or status so that in webhooks the response will have the accurate information. Correct me if I'm wrong but `Response.OK({ success: true, message: "Message sent!" });` will be sent even if this POST fails

Reply:
Not familiar with the Klavio API, but if i got it right, we could add even the course of the error from the message field (https://developers.klaviyo.com/en/reference/api-overview#errors). Might be useful during debugging